app/scripts/directives/schoollist.coffee

```diff
@@ -20,12 +20,13 @@ angular.module 'edudashApp'
       property: '@property'
       max: '=max'
       min: '=min'
```

Comment:
cool, looks great! Small follow-up comment on that comment for the `UNKNOWN` case